src/Deposit.js

```diff
@@ -523,6 +533,21 @@ export default class Deposit {
     // Bitcoin txHash; no verification initially.
   }
 
+  /**
+   * Registers a handler for notification when the Bitcoin funding transaction
+   * has received a confirmation
+   *
+   * @param {OnReceivedConfirmationHandler} onReceivedConfirmationHandler
+   *        A handler that passes an object with the transactionID and
+   *        confirmations as its parameter
```

Comment:
Periods on both of these sentences please :)

I would also say the handler receives rather than passes those parameters, from the caller's perspective.

Reply:
Addressed!